test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -53,6 +52,28 @@ class IntervalCoverageSender(
         job.cancel()
         logger.debug { "Coverage sending job is stopped." }
     }
+
+    /**
+     * Create a function which sends chunks of test coverage to the admin part of the plugin
+     * @return the function of sending test coverage
+     * @features Coverage data sending
+     */
+    private fun sendProbes(data: Sequence<ExecDatum>) {
+        if (!coverageTransport.isAvailable().get()) {
```

Comment:
An idea (_To be discussed_): should we move this check and buffering to the very end of the data mapping/sending chain.
Possible issues / overheads:
- serialized + compressed data vs actual classes - RAM consumption
Pros: simpler code, less juggling with transport availability